azurerm/resource_arm_kubernetes_cluster_test.go

```diff
@@ -98,6 +98,33 @@ func TestAccAzureRMKubernetesCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMKubernetesCluster_aadProfile(t *testing.T) {
+	resourceName := "azurerm_kubernetes_cluster.test"
+	ri := acctest.RandInt()
+	clientId := os.Getenv("ARM_CLIENT_ID")
+	clientSecret := os.Getenv("ARM_CLIENT_SECRET")
+	serverAppId := os.Getenv("ARM_SERVER_APP_ID")
+	serverAppSecret := os.Getenv("ARM_SERVER_APP_SECRET")
+	clientAppId := os.Getenv("ARM_CLIENT_APP_ID")
```

Comment:
How can we set these environment variables?

Reply:
I don't understand what you mean here. Could you elaborate?

Reply:
The current doc at https://www.terraform.io/docs/providers/azurerm/ does not show how to create app_id and app_secret. We may consider to add it.